include/glow/CodeGen/MemoryAllocator.h

```diff
@@ -156,8 +241,27 @@ class MemoryAllocator {
   /// candidates.
   void evictFirstFit(uint64_t size, const std::set<Handle> &mustNotEvict,
                      std::vector<Handle> &evicted);
+
   /// Associates a \p handle with an allocated address \p ptr and size \p size.
   void setHandle(uint64_t ptr, uint64_t size, Handle handle);
+
+  /// Function to verify the allocations before allocating the segments with
+  /// \ref allocateAll. \returns true if allocations are valid and false
+  /// otherwise.
+  bool verifyAllocations(const std::list<Allocation> &allocList) const;
+
+  /// Function to verify the segments allocated with \ref allocateAll.
```

Comment:
Could you describe the parameters of the function in the comment?

Reply:
Done.